Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/BaseWebServiceTest.java

```diff
@@ -22,12 +21,12 @@
   private ThreadLocal<WebServiceTestObject> webServiceTestObject = new ThreadLocal<WebServiceTestObject>();
```

Comment:
Granted, this is just for the rename, but we could hit this with a deprecation tag now or later.  Either way works.

Reply:
Probably worth just getting it in there now.